server/db/prisma/schema.prisma

```diff
@@ -11,16 +11,41 @@ datasource db {
 }
 
 model User {
-  id          String @id @default(uuid())
-  name        String
-  email       String
-  createdAt   DateTime @default(now())
-  updatedAt   DateTime @default(now())
+  id        String   @id @default(uuid())
+  name      String
+  email     String
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now())
+  School    School[]
 }
 
 model School {
-  id          String @id @default(uuid())
-  name        String
-  createdAt   DateTime @default(now())
-  updatedAt   DateTime @default(now())
+  id             String         @id @default(uuid())
+  name           String
+  country        String
+  location       String
+  responsible    User           @relation(fields: [responsibleId], references: [id])
+  responsibleId  String
+  deploymentDate DateTime
+  createdAt      DateTime       @default(now())
+  updatedAt      DateTime       @default(now())
+  schoolSurveys  SchoolSurvey[]
+}
+
+model Programme {
```

Comment:
Berkeli, just to confirm.
Is 'programme' part of the final schema?
The reason I ask is because I saw it on the snapshot 
But I did not see it on the 'data model' you posted in Slack

Reply:
Well it's not confirmed by the client, but it is in the spreadsheet and it is a key column when they entered activity information (monthly/quarterly surveys)